contrib/monitoring-mixins/jsonnetfile.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": 1,
+  "dependencies": [
+    {
+      "source": {
+        "git": {
+          "remote": "https://github.com/grafana/grafonnet-lib.git",
+          "subdir": "grafonnet"
+        }
+      },
+      "version": "8fb95bd89990e493a8534205ee636bfcb8db67bd"
+    }
+  ],
+  "legacyImports": true
```

Comment:
Since this is a new mixins, it might be good to start without legacy stuff :)

Reply:
Ah, this is what [`mixtool generate`](https://github.com/monitoring-mixins/mixtool#generate) gave me. Can I just flip this boolean or is there anything else I should cross-update too?

Reply:
Self-answer: yes, that seems to be enough.